css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: digital;
+  src: url(fonts/digital-7.ttf);
+}
+
+
+
+body {
+    background: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+input {
```

Comment:
avoid styling by tags, better to style via classes.

Reply:
Got it. Shall I give class name to "body" tag too?